scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -50,33 +56,34 @@ function doSetup() {
 
 function doCheckDockerVersions() {
   local DOCKERENGVERSION
-  DOCKERENGVERSION=$(docker version | grep "Version" | head -1 | sed -ne 's/.*\([0-9][0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+  DOCKERENGVERSION=$(docker version --format '{{.Client.Version}}' | sed -ne 's/.*\([0-9][0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
```

Comment:
this is outputting nothing but the version. So what is the need of the `sed` command?